core/server/models/post.js

```diff
@@ -157,7 +166,7 @@ Post = ghostBookshelf.Model.extend({
                 return Promise.reject(new errors.ValidationError(
                     i18n.t('errors.models.post.expectedPublishedAtInFuture')
                 ));
-            } else if (moment(publishedAt).isBefore(moment().add(5, 'minutes'))) {
+            } else if (this.hasChanged('published_at') && moment(publishedAt).isBefore(moment().add(2, 'minutes'))) {
```

Comment:
This is a guard with an arbitrary number... should it be defined in config somewhere, even if it isn't overridable?

Some more CASE comments here would be handy - this one in particular is quite hard to read